Source/Core/Core/HW/DVD/DVDThread.cpp

```diff
@@ -386,15 +386,15 @@ static void PatchReadRequest(std::vector<u8>& buffer, u64 offset)
       u64 start = it->first - offset;
       std::memmove(buffer.data() + start, it->second.data(),
                    std::min(it->second.size(), buffer.size() - start));
-      INFO_LOG(DVDINTERFACE, "patch applied at %08lx", offset + start);
+      INFO_LOG(DVDINTERFACE, "patch applied at %08llux", offset + start);
```

Comment:
You should use `PRIx64` for `u64`.

Reply:
why? 

Current versions of Visual studio support `%llux`

Reply:
`%llu` works for `u64` without warnings on all compilers now? I guess you could use that then.